products/wagtail.md

```diff
@@ -11,27 +11,33 @@ auto:
 -   git: https://github.com/wagtail/wagtail.git
 sortReleasesBy: releaseDate
 releases:
+#-   releaseCycle: "4.0"
+#    lts: false
+#    latest: "N/A"
+#    releaseDate: 2022-09-01
+#    support: 2023-02-01
+#    eol: 2023-02-01
 -   releaseCycle: "3.0"
-    support: 2022-08-01
     lts: false
-    eol: false
     latest: "3.0.1"
-    latestReleaseDate: 2022-06-16
     releaseDate: 2022-05-16
+    latestReleaseDate: 2022-09-01
```

Comment:
Can you confirm where this is from? The date [here](https://github.com/wagtail/wagtail/releases/tag/v3.0.1) is 16th June, and this is in the future.